src/index.ts

```diff
@@ -2723,9 +2791,311 @@ export default class Decimal {
     return lnx.div(lnx.lambertw());
   }
 
+  //Super-root, one of tetration's inverses - what number, tetrated to height (height), equals this?
```

Comment:
Wow, this saves me a lot of work, nice one!

The unit test looks robust at testing this. I tried a variation of the unit test, but all it told me was that if tetrating a small number returns 1 then obviously sroot can no longer tell you what small number you originally had (you'd need a format of number that's like '1 plus or minus a very tiny amount', which is not something that a number go up library needs to support, though an arbitrary precision library would consider it).

Reply:
Super root was the main reason I made all this - I really wanted it included. (Adding a linear approximation argument to tetration was my second reason for making this)